docs/Liquidity Provider Guide.md

```diff
@@ -0,0 +1,393 @@
+
+    bitcoin testnet 3, litecoin testnet 4, ethereum rinkeby. Faucets: [t-BTC](https://coinfaucet.eu/en/btc-testnet/), [t-LTC](https://testnet.help/en/ltcfaucet/testnet), [t-ETH 1](https://faucet.rinkeby.io/) or [2](https://testnet.help/en/ethfaucet/rinkeby). If you need help or some testnet coins, hit us up on [Discord](https://discord.gg/RnXFHpn)!
+
+3. **Mainnet**. `Status: down | Required CPUs: 2 for light, 4 for full | RAM: 2GB for light, 16GB for full | Disk: 1GB for light, 1TB for full | Initial Sync Time: 30 mins for light, 72h for full`
+    
+   Down until all breaking changes are merged and some weeks on testnet didn't reveal major issues.
+
+
+## Hardware
+Since liquidity providers should be online 24/7 and we are ushering in a post-cloud era, we recommend setting up a power-efficient linux box connected to your router. No special configurations, like port forwardings, are necessary. Running your opendexd setup in the cloud is obviously possible, just not something we encourage to do.
+
+[**🧑‍🏭 Standard Hardware Guide**](Standard%20Hardware%20Guide.md): This guide walks you through setting up an arm64-based Raspberry Pi3/4. Costs: **65€-290€**
+
+[**💪 Pro Hardware Guide**](Pro%20Hardware%20Guide.md): This guide walks you through setting up a powerful amd64-based Mini PC. Costs: **180€-465€**
+
+🎚️ **Custom**: If you are using a different device or a cloud VPS:
+  * Check the hardware requirements for the different networks and modes above
+  * The full setup requires a SSD for geth being able to sync. For the light setup, a regular HDD/SD card is fine.
+  * If you are using a VPS for testnet or mainnet, you can switch to 2 cores & 4 GB RAM after initial sync, given you use default settings.
+  * We currently support `amd64` (also called `x86`/`x64`) and `arm64` (also called `aarch64`/`armv8`), which should cover most devices and services.
+
+## Software
+
+Docker & Docker Compose.
+
+Version >= 18.09 on Linux or Windows 10 [using WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10). If you do not have docker & docker-compose installed yet and you are using Ubuntu 20.04 LTS, install these via `sudo apt install docker.io`. If you are using any version besides Ubuntu 20.04, follow the official [docker install instructions](https://docs.docker.com/get-docker/). Also make sure that the current user can run docker commands. Test with `docker run hello-world`. If this fails, [follow these instructions](https://docs.docker.com/engine/install/linux-postinstall/). This guide was written using Ubuntu 20.04 LTS.
+
+# The Setup
+
+From here we assume that your device is running with docker set up. Check the guides in the hardware section above if your device is not ready yet.
+
+## Let's Roll
+
+Start the environment with
+
+```bash
+curl https://raw.githubusercontent.com/opendexnetwork/opendex-docker/master/opendexd.sh -o ~/opendexd.sh
```

Comment:
Do we want to upgrade these to use the new launcher binary instead?

Reply:
Or hmm... launcher currently does not have interactive mode if I'm not mistaken. :thinking:

Reply:
> Do we want to upgrade these to use the new launcher binary instead?

Yes.

> Or hmm... launcher currently does not have interactive mode if I'm not mistaken. thinking

We'll need this rather sooner than later. I quite some situations I can't access any webapp, e.g. setting up opendex-docker on a VPS. Can we bring back interactive mode for create flow and usage via CLI? @Maith8Ae

Reply:
I'd expect this to not be super quick effort. Are we still `opendexd.sh` compatible for interactive mode @Maith8Ae?

If so, perhaps we can merge as is and do a follow-up PR once the launcher interactive mode is ready. This PR is huge and quite difficult to review as it covers a lot.

Reply:
Agreed, add a WIP disclaimer which we can remove once interactive CLI mode is back: https://opendex.network/v/develop/docs/liquidity-provider-guide#lets-roll-wip-not-fully-working-yet